plugins/filebrowser/tpl/editorbrowser.html

```diff
@@ -5,7 +5,7 @@
 <html lang="en">
     <head>
         <title>Filebrowser</title>
-        <link rel="stylesheet" href="<?php echo "{$pth['folder']['corestyle']}stylesheet.css"?>" type="text/css">
+        <link rel="stylesheet" href="<?php echo "{$pth['folder']['corestyle']}xhstyles.css"?>" type="text/css">
         <style type="text/css">body {padding: 0; margin: 0; overflow: auto;}</style>
```

Comment:
Oops – now I got it. It's a simple bug fix, that can be committed right away (done in fe4517b).